kivy/logger.py

```diff
@@ -115,8 +116,8 @@ def purge_logs(self, directory):
         if randint(0, 20) != 0:
             return
 
-        # Use config ?
-        maxfiles = 100
+        from kivy.config import Config
+        maxfiles = Config.getint('kivy', 'log_maxfiles') - 1
```

Comment:
The question is the meaning of *log_maxfiles*. With the current implementation, you get one logfile if you specify log_maxfiles=1 in config.ini (purge removes all old logfiles and the logger then adds a new one). Without the "-1" you get two logfiles with the log_maxfiles=1 setting - which one is better?